dotCMS/src/main/java/com/dotcms/cube/PaginationCubeJSResultSet.java

```diff
@@ -0,0 +1,77 @@
+package com.dotcms.cube;
+
+import com.dotcms.util.DotPreconditions;
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+/**
+ * It represent un {@link CubeJSResultSetImpl} with pagination, it is going to get the result by a limit
```

Comment:
It represents a*

Reply:
Should't it be CubeJSResultSet instead of the Impl?

Reply:
done https://github.com/dotCMS/core/pull/24881/commits/e5473ac3f005d3c215fe0aa4bbe2145613d8fdc8